addon/components/liquid-container.js

```diff
@@ -2,15 +2,14 @@ import Component from '@ember/component';
 import Growable from 'liquid-fire/mixins/growable';
 import { measure } from './liquid-measured';
 import layout from 'liquid-fire/templates/components/liquid-container';
-import $ from 'jquery';
 
 export default Component.extend(Growable, {
   layout,
   classNames: ['liquid-container'],
 
   lockSize(elt, want) {
-    elt.outerWidth(want.width);
-    elt.outerHeight(want.height);
+    elt.style.width = `${want.width}px`;
+    elt.style.height = `${want.height}px`;
```

Comment:
It looks like `elt` in this case was a jQuery-wrapped instance (of the element), so will `elt` now be an actual DOM element instead?

Reply:
This is neither part of these changes, nor the right repository. This was meant to be a PR against a fork.